### PR TITLE
react-dropzone: export Dropzone by default as it is in the original package

### PR DESCRIPTION
**Pull request by Lodin**
From [sources](https://github.com/react-dropzone/react-dropzone/blob/master/src/index.js#L417) `Dropzone` should be exported by default, not using `export = Dropzone`. It is also actual for the `4.1.3` version ([url](https://github.com/react-dropzone/react-dropzone/blob/1d270e5379576c1e62cb7ae6aba905f31a372c02/src/index.js#L417)). 

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [current version](https://github.com/react-dropzone/react-dropzone/blob/master/src/index.js#L417), [version 4.1.3](https://github.com/react-dropzone/react-dropzone/blob/1d270e5379576c1e62cb7ae6aba905f31a372c02/src/index.js#L417)
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by RyanCavanaugh** [comment]
This PR has been open and unchanged 5 days without signoff or complaint. This will be merged by a maintainer soon if there are no objections.

**Review by benbayard — CHANGES_REQUESTED** [review]
This is not really a default export. They use a webpack plugin to make this look like a default export. If there was a PR on dropzone that changed that can you please link it here?

**Review by benbayard — CHANGES_REQUESTED** [review]
image
This is not a default export, this cannot be merged.

**Comment by benbayard** [comment]
@RyanCavanaugh It won't let me block merge, but this should not be merged in.

**Comment by Lodin** [comment]
@benbayard Thanks for explanation, didn't think about the plugin.

I guess the problem is laying in [this commit](https://github.com/react-dropzone/react-dropzone/commit/1fd768b2a95d53dadc8c746cb83a373bbe312ab5). It adds "module" property at [package.json](https://github.com/react-dropzone/react-dropzone/blob/master/package.json#L5) that redirects ES6 import to a `dist/es/index.js` that has default export, and the application webpack building fails. 

Not sure what is the best way to handle it. ES-module based building fails (that was a trigger forced me to prepare this PR), but I see that my changes does not cover the whole package. What do you think?

**Comment by benbayard** [comment]
You have 2 options: 
1. Use a commonjs import as in the TS example
2. You can use synthetic default exports in your tsconfig

**Comment by Lodin** [comment]
@benbayard Thanks, the second one works

**Pull request closed without merging**
